.travis.yml

```diff
@@ -1,5 +1,10 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.4.3
-before_install: gem install bundler -v 1.16.0
+  - 2.3.2
+before_install:
+  - gem install bundler -v 1.16.1
+script:
+  - RAILS_ENV=test bundle exec rake spec
+  - bundle exec appraisal install && RAILS_ENV=test bundle exec appraisal rails-5 cucumber
```

Comment:
Thoughtbot recommends to use multiple gemfiles: https://github.com/thoughtbot/appraisal#travis-ci-integration

Reply:
fixed